src/include/cell.i

```diff
@@ -681,11 +764,14 @@ restart:
      * following switch. All validity windows default to durability.
      */
     unpack->v = 0;
+    unpack->durable_start_ts = WT_TS_NONE;
+    unpack->durable_stop_ts = WT_TS_MAX;
     unpack->start_ts = WT_TS_NONE;
     unpack->start_txn = WT_TXN_NONE;
     unpack->stop_ts = WT_TS_MAX;
     unpack->stop_txn = WT_TXN_MAX;
-    unpack->newest_durable_ts = WT_TS_NONE;
+    unpack->oldest_durable_ts = WT_TS_NONE;
+    unpack->newest_durable_ts = WT_TS_MAX;
```

Comment:
As mentioned above `WT_TS_NONE` would be better for `newest_durable_ts` also.

Reply:
According to the discussion summary in the ticket, we should follow the default values of `WT_TS_NONE` for all durable timestamps irrespective of whether it is `start` or `stop` for both value and address cells.

Reply:
Fixed.